src/js/constants/StoreConfig.js

```diff
@@ -196,6 +199,13 @@ const ListenersDescription = {
     listenAlways: false
   },
 
+  sidebar: {
```

Comment:
every single store has `listenAlways` and `unmountWhen` set. can you just default them to `true` and a function that returns true, respectively?